tests/security-test.py

```diff
@@ -0,0 +1,111 @@
+        cluster_template_without_pod_security = CONST.REPO_ROOT.joinpath("tests/config/ray-cluster.mini.yaml.template")
+        helm_chart_values_yaml_template = CONST.REPO_ROOT.joinpath("helm-chart/kuberay-operator/values.yaml")
+        helm_chart_values_yaml = CONST.REPO_ROOT.joinpath("ray-operator/config/security/security-values.yaml")
+        image_dict = {
+            CONST.RAY_IMAGE_KEY: ray_image,
+            CONST.OPERATOR_IMAGE_KEY: kuberay_operator_image
+        }
+        cluster_namespace = "pod-security"
+
+        # copy helm-chart/kuberay-operator/values.yaml and update the field securityContext in the new security-values.yaml
+        with open(helm_chart_values_yaml_template) as base_values:
+            with open(helm_chart_values_yaml,'w') as security_values:
+                base_values = yaml.load(base_values, Loader=yaml.FullLoader)
+                base_values['securityContext']['allowPrivilegeEscalation'] = False
+                base_values['securityContext']['capabilities'] = {'drop':["ALL"]}
+                base_values['securityContext']['runAsNonRoot'] = True
+                base_values['securityContext']['seccompProfile'] = {'type': 'RuntimeDefault'}
+                yaml.dump(base_values,security_values)
+
+        K8S_CLUSTER_MANAGER.delete_kind_cluster()
+        K8S_CLUSTER_MANAGER.create_kind_cluster(kind_config = CONST.REPO_ROOT.joinpath("ray-operator/config/security/kind-config.yaml"))
+
+        # apply the restricted Pod security standard to all Pods in the namespace pod-security. 
+        # The label pod-security.kubernetes.io/enforce=restricted means that the Pod that violate the policies will be rejected 
+        shell_subprocess_run(f"kubectl create ns {cluster_namespace}")
+        time.sleep(1) 
+        shell_subprocess_run( f" kubectl label --overwrite ns {cluster_namespace} \
+                                {cluster_namespace}.kubernetes.io/warn=restricted \
+                                {cluster_namespace}.kubernetes.io/warn-version=latest \
+                                {cluster_namespace}.kubernetes.io/audit=restricted \
+                                {cluster_namespace}.kubernetes.io/audit-version=latest \
+                                {cluster_namespace}.kubernetes.io/enforce=restricted \
+                                {cluster_namespace}.kubernetes.io/enforce-version=latest" )
+      
+        # Install the KubeRay operator in pod-security namespace and create a RayCluster with proper securityContext configurations
+        time.sleep(1) 
+        operator_manager = OperatorManager(image_dict)
+        operator_manager.prepare_operator(namespace=cluster_namespace,helm_chart_values = helm_chart_values_yaml)
+        utils.create_ray_cluster(cluster_template_with_pod_security, ray_version, ray_image, namespace=cluster_namespace)                        
+   
+def parse_environment():
```

Comment:
This test is not a compatibility test. We do not need to run this test with different versions of Ray and KubeRay. Hence, we can remove this function.